src/Alert.js

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from "react";
+import { Alert } from "react-bootstrap";
+import { ErrorIcon } from "./resources/ErrorIcon";
+
+export const Alerts = (props) => {
+  useEffect(() => {
+    // Focus on the alert the first time it's rendered--maybe there's a better way to do this and you can teach me
```

Comment:
I've never done this before but I googled it and apparently this is the way to do it :\